### PR TITLE
🚩 Resolve #1047

### LINKED ISSUE #1047: Controlling slider title properties
I'm trying to control the font size of a slider widget title, but it seems to be directly tied to the window size and not controlled by anything in rcParams. I'm able to alter other parameters of the title and label directly (see below), but not the font size. 

Would expect to be able to alter the font size of each slider. I also wonder if I'm missing a more simple interface to slider title properties (within the `add_slider_widget` call perhaps).

(Edited for markdown formatting).

-----

**To Reproduce**

Please include a code snippet to reproduce the bug in the block below:

```py
# Insert code here

p = pv.Plotter()

def create_mesh(value):
    res = int(value)
    sphere = pv.Sphere(phi_resolution=res, theta_resolution=res)
    p.add_mesh(sphere, name='sphere', show_edges=True)
    return

slider = p.add_slider_widget(create_mesh, [5, 100], title='Resolution')

# Can alter these
slider.GetRepresentation().GetTitleProperty().SetOpacity(0.5)
slider.GetRepresentation().GetTitleProperty().SetColor((1.0,0.0,0.0))
slider.GetRepresentation().SetLabelFormat('%0.9f')

# Can't alter this
slider.GetRepresentation().GetTitleProperty().SetFontSize(1000)

p.show()
```

**Screenshots**
slider_title

-----

**System Information:**
 Please run the following code wherever you are experiencing the bug and paste the output below. This report helps us track down bugs and it is critical to addressing your bug:

```py
# Get system info
import pyvista as pv
print(pv.Report())
```

```txt
# Paste system info here
--------------------------------------------------------------------------------
  Date: Mon Dec 07 18:32:10 2020 EST

                OS : Darwin
            CPU(s) : 4
           Machine : x86_64
      Architecture : 64bit
       Environment : IPython
        GPU Vendor : Intel Inc.
      GPU Renderer : Intel Iris OpenGL Engine
       GPU Version : 4.1 INTEL-14.5.22

  Python 3.8.6 | packaged by conda-forge | (default, Oct  7 2020, 18:42:56)
  [Clang 10.0.1 ]

           pyvista : 0.27.2
               vtk : 9.0.1
             numpy : 1.19.4
           imageio : 2.9.0
           appdirs : 1.4.4
            scooby : 0.5.6
            meshio : 4.3.6
        matplotlib : 3.3.3
         pyvistaqt : 0.2.0
             PyQt5 : 5.12.3
           IPython : 7.19.0
             scipy : 1.5.3
--------------------------------------------------------------------------------
```

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

 

We can plot following (resolve #1047):
```python
import pyvista as pv

p = pv.Plotter()


def create_mesh(value):
    res = int(value)
    sphere = pv.Sphere(phi_resolution=res, theta_resolution=res)
    p.add_mesh(sphere, name="sphere", show_edges=True)
    return


slider = p.add_slider_widget(
    create_mesh,
    [5, 100],
    title="Resolution",
    title_opacity=0.5,
    title_color="red",
    fmt="%0.9f",
    title_height=0.08,
)

p.show()
```
slider_widget

### Details

- None

### CODE REVIEW

File: pyvista/plotting/widgets.py
Comment (akaszynski) [inline pyvista/plotting/widgets.py:737]:
```suggestion

        title_height: float, optional
            Relative height of the title as compared to the length of the slider.

        title_opacity: str, optional
            Opacity of title. Defaults to 1.0.

        title_color : string or 3 item list, optional
            Either a string, rgb list, or hex color string.  Defaults to the value 
            given in ``color``.

        fmt : str, optional
            String formatter used to format numerical data. Defaults to ``None``.
```

**Review by akaszynski — APPROVED** [review]
LGTM.  Minor doc fixes.  Would also be nice to see an example:

```Examples
--------
...
```

**Review by akaszynski — APPROVED** [review]
Great example!  Thanks for adding it.

I've also sent an invitation for you to be part of the developers group since you're regularly active on this repo and I'd like to give you merge permissions.  Once you've accepted the request you'll be able to merge PRs that have approvals from the dev team.  You'll also be able to review PRs as well, though you don't have to and I'd be careful about approving PRs without a through review (I know that's bitten me a few times).

Cheers!

**Comment by tkoyama010** [comment]
@akaszynski Thank you for invitation. It's my honor.

**Pull request merged**
